src/data/format-links.ts

```diff
@@ -20,7 +20,9 @@ export const formatLinks: Dict<IFormatDataLinks> = {
 	'gen9lc': {
 		teams: 'https://www.smogon.com/forums/threads/3712989/',
 	},
-
+	'gen9ubers': {
+		teams: 'https://www.smogon.com/forums/threads/3719619/'.
```

Comment:
```suggestion
		teams: 'https://www.smogon.com/forums/threads/3719619/',
```

I told on PS but seems you forgetting